commands/version.go

```diff
@@ -40,16 +41,20 @@ func runVersion(cmd *cobra.Command, args []string) {
 	if shortVersion {
 		fmt.Println(version.BuildVersion())
 	} else {
-		figletColoured := aec.BlueF.Apply(figletStr)
-		if runtime.GOOS == "windows" {
-			figletColoured = aec.GreenF.Apply(figletStr)
-		}
-		fmt.Printf(figletColoured)
+		printFiglet()
 		fmt.Printf("Commit: %s\n", version.GitCommit)
 		fmt.Printf("Version: %s\n", version.BuildVersion())
 	}
 }
 
+func printFiglet() {
+	figletColoured := aec.BlueF.Apply(figletStr)
+	if runtime.GOOS == "windows" {
+		figletColoured = aec.GreenF.Apply(figletStr)
+	}
+	ansi.Printf(figletColoured)
```

Comment:
Thanks for the change. I just don't understand why this stopped working.. We also have to test this on Linux and Mac before merging.

Reply:
@alexellis CI breaks because I did not commit vendored files

Reply:
It works propely with linux @itscaro . Just if you can add the dependency it would be great.